deb/VERSION

```diff
@@ -0,0 +1 @@
+1.10
```

Comment:
seeing `1.8` below, should they be in sync?

Reply:
No they've been out of sync a while now.